CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+- proxy support tests
```

Comment:
This needs a verb really to be consistent with the rest of this, like "Added tests for proxy support"